### PR TITLE
fix lingering issues from #57 following @timelyportfolio's approach

### PR DESCRIPTION
**Pull request by joshhjacobson**
I was still having issues when an array of axes that included the rightmost axis was hidden. This gave the error `TypeError: Cannot read property 'type' of undefined` from this line in `brushFor.js`:
```javascript
brushRangeMax = config.dimensions[axis].type 
```
I followed the same approach released in [v2.2.5](https://github.com/BigFatDog/parcoords-es/releases/tag/v2.2.5) to correct this.

### CODE REVIEW

**Comment by joshhjacobson** [comment]
@BigFatDog sorry to bother you, but is there any way we can get this PR merged in the next few days? I have a deadline coming up at the end of the month and it would be great to have this fixed by then.

**Comment by BigFatDog** [comment]
@joshhjacobson 
Don't worry. I do it right now.

**Comment by joshhjacobson** [comment]
My bad, I didn't realize you'd already completed a review. Where can I find that?

**Comment by BigFatDog** [comment]
Oh MY GOD! I didn't submit it... 

Sorry, my bad.

File: src/brush/1d/brushFor.js [hunk 1]
```diff
@@ -13,12 +13,16 @@ const brushFor = (state, config, pc, events, brushGroup) => (
...
   _selector
 ) => {
-  const brushRangeMax =
-    config.dimensions[axis].type === 'string'
-      ? config.dimensions[axis].yscale.range()[
-          config.dimensions[axis].yscale.range().length - 1
-        ]
-      : config.dimensions[axis].yscale.range()[0];
+  // handle hidden axes which will not be found in dimensions
+  let brushRangeMax = null;
```
[the remark this answers is not in the document]
Comment (joshhjacobson) [reply src/brush/1d/brushFor.js]: Thanks for the feedback, I see what you mean. I'll implement this change right now.

[the remark this answers is not in the document]
Comment (BigFatDog) [reply src/brush/1d/brushFor.js]: I'll be online to make sure this is merged tonight

[the remark this answers is not in the document]
Comment (joshhjacobson) [reply src/brush/1d/brushFor.js]: Thanks so much!

[the remark this answers is not in the document]
Comment (BigFatDog) [reply src/brush/1d/brushFor.js]: Sorry again for my mistake. I wish it didn't delay your work.

[the remark this answers is not in the document]
Comment (joshhjacobson) [reply src/brush/1d/brushFor.js]: To clarify, did you want to take the following approach:
```javascript
if (!config.dimensions.hasOwnProperty(axis)) {
  return () => {}
}

// no more check
const brushRangeMax = ...
const yScale = ...
```
which leaves the code something like this:
```javascript
// handle hidden axes which will not be a property of dimensions
if (!config.dimensions.hasOwnProperty(axis)) {
 return () => {}
}

const brushRangeMax =
    config.dimensions[axis].type === 'string'
      ? config.dimensions[axis].yscale.range()[
          config.dimensions[axis].yscale.range().length - 1
        ]
      : config.dimensions[axis].yscale.range()[0];

const yscale = config.dimensions[axis].yscale;
```
i.e. change what @timelyportfolio had done for #57?

[the remark this answers is not in the document]
Comment (BigFatDog) [reply src/brush/1d/brushFor.js]: keeping @timelyportfolio's fix unchanged. 

Only adding this to the function:
```
const brushFor = (state, config, pc, events, brushGroup) => (
  axis,
  _selector
) => {
  // add this line
  if (!config.dimensions.hasOwnProperty(axis)) {
      return () => {}
  }
  // the rest code remain unchanged.
  const brushRangeMax = ....
```

[the remark this answers is not in the document]
Comment (joshhjacobson) [reply src/brush/1d/brushFor.js]: Got it

**Comment by timelyportfolio** [comment]
@joshhjacobson @BigFatDog thanks so much.  Sorry I missed that!

**Comment by joshhjacobson** [comment]
@timelyportfolio no problem! Wouldn't have been able to figure this out without your initial fix.

**Comment by BigFatDog** [comment]
> Much appreciated!
You're welcome.

**Pull request merged**